### PR TITLE
fix title error on new notebook

### PR DESCRIPTION
**Pull request by alaq**
Running a new notebook returned an error as there was no title on the new notebooks.

Addresses #315

### CODE REVIEW

**Comment by ry** [comment]
Thanks Adrien! Could we get a test for this to prevent the error in the future?

File: website/db.ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ class DatabaseFB implements Database {
     const docRef = nbCollection.doc(nbId);
     await docRef.update({
       cells: doc.cells,
-      title: doc.title,
+      title: doc.title || "",
```
Comment (qti3e) [inline website/db.ts]: Shouldn't it be `Untitled notebook`?

**Comment by alaq** [comment]
Will add the test now.

File: website/db.ts (hunk 1 above)
Comment (alaq) [reply website/db.ts]: If the title is "", it gets replaced with a grey "Untitled Notebook". And we want to display something, even when the title is empty.

**Comment by ry** [comment]
@alaq Actually I realize now that we don't have any tests that run against the actual database (only the mock database), which is required to properly test this error. I've added an issue #321 to address this. 

Please squash this commits and I will land them as is.

File: website/notebook.ts [hunk 2]
```diff
@@ -610,7 +610,7 @@ export class Notebook extends Component<NotebookProps, NotebookState> {
 
       const titleDisplay = h("div", { class: "title" }, [
         h("h2", {
-          class: doc.title != null ? "" : "untitled",
+          class: doc.title && doc.title.length ? "" : "untitled",
```
Comment (alaq) [inline website/notebook.ts]: I added this check here since we also want the title greyed out if `doc.title === ""`.

**Pull request merged**